Source/Core/UICommon/GameFile.h

```diff
@@ -44,6 +44,13 @@ bool operator!=(const GameBanner& lhs, const GameBanner& rhs);
 class GameFile final
 {
 public:
+  enum class Variant
+  {
+    LongAndPossiblyCustom,
+    LongAndNotCustom,
+    ShortAndNotCustom,
```

Comment:
This will be the nittiest of nits, but...thoughts on using `LongButNotCustom`, `ShortButNotCustom`? :)
But then again, should it be `LongOrPossiblyCustom` for the first one?

You should probably ignore this comment for the sake of all the sheds we built so far.

Reply:
Using `But` there doesn't make too much sense to me, since there's no logical opposition between `Long` and `Custom` in `LongButNotCustom`. Also, it feels slightly awkward to make it inconsistent by using `But` for the last two ones but not the first one.